timewsync/__init__.py

```diff
@@ -186,7 +186,7 @@ def sync(configuration: Configuration) -> None:
 
     # Active time tracking
     if active_interval:
-        print("Time tracking is active. Stopped and restarted time tracking to prevent conflicts.", file=sys.stderr)
+        print("Time tracking is active. Stopped time tracking to prevent conflicts.")
```

Comment:
I believe we originally settled on printing stuff meant for humans to `stderr` and leaving `stdout` for strictly formatted things.

Reply:
I agree to some extend.
According to [this stackoverflow discussion](https://stackoverflow.com/questions/35461687/when-to-use-sys-stdout-instead-of-sys-stderr), `stdout` is for the _main_ output, i.e. the essence of the program.
Therefore I would include the `Synchronization successful!` message, which is unlikely to change in the future (unless we switch to British english).

The other info concerning active intervald could change, as it did recently.
It is also not as important, since we only split an interval into two at the point of synchronization.
Therefore I would suggest to revert those ones but keep the main print in `stdout`.

Reply:
```suggestion
        print("Time tracking is active. Stopped time tracking to prevent conflicts.", file=sys.stderr)
```

Reply:
```suggestion
        print("Time tracking is active. Stopped time tracking to prevent conflicts.", file=sys.stderr)
```

Reply:
Good catch! What should we use instead? Maybe `log.info`?

Reply:
Counterpoint: When using timewsync in a script / piping context, the exit code should already indicate success or failure. Therefore, the "Synchronization successful!" message does not seem to add real value.

But I don't have a enough experience with Unix scripting and piping to form a strong opinion on this and would be fine either way.

Reply:
I have read up on it and it seems to be a serious debate across developers.

Since we only return progress messages I agree that we should not print them to `stdout` as it doesn't offer valuable information to a pipeline.

I also think that `stderr` is not a suitable alternative, since we already use logging and could easily write it to `log.info`, as @joachimschmidt557 proposed.

For this PR I would like to revert to `stderr` first, since I would like to revise the whole logging part (we only debug/info log errors, which could be expanded).